exporter/datadogexporter/examples/k8s-chart/daemonset.yaml

```diff
@@ -22,7 +22,7 @@ spec:
         command:
           - "/otelcol-contrib"
           - "--config=/conf/otel-agent-config.yaml"
-        image: otel/opentelemetry-collector-contrib:0.54.0
+        image: otel/opentelemetry-collector-contrib:0.55.0
```

Comment:
This should at least be 0.57.0 to incorporate the breaking changes we made to the configuration and elsewhere

Reply:
Updated to 0.59.0